src/Database/PostgreSQL/Simple/Migration.hs

```diff
@@ -116,12 +157,19 @@ sequenceMigrations = \case
 
 -- | Executes all SQL-file based migrations located in the provided 'dir'
 -- in alphabetical order.
-executeDirectoryMigration :: Connection -> Bool -> FilePath -> IO (MigrationResult String)
-executeDirectoryMigration con verbose dir =
+executeDirectoryMigration
+    :: LogWrite
+    -> Connection
+    -> Bool
+    -> FilePath
+    -> IO (MigrationResult String)
+executeDirectoryMigration logWrite con verbose dir =
     scriptsInDirectory dir >>= go
     where
         go fs = sequenceMigrations (executeMigrationFile <$> fs)
-        executeMigrationFile f = executeMigration con verbose f =<< BS.readFile (dir ++ "/" ++ f)
```

Comment:
It looks like you ran some kind of autoformatter to the source? *I* don't mind and if were the maintainer I would pass this, but maybe be on the safe side and avoid autoformatters on unfamiliar codebases? Formatting can be a touchy subject for some.

Reply:
No, I didn’t use no formatter. All changes are manual. I realized almost all the lines in the code are limited to 80 chars. But only some aren’t. So I split those lines into few.

Reply:
I'm still hesitant on it. These kinds of extra changes are just extra noise for the reviewer/maintainer.

Reply:
I had to touch this line anyway.